liquidity-module/index.md

```diff
@@ -131,7 +160,7 @@ For the testnet `target` chain, use the following values.
 
 - Target Gas Price (0.025uatom): 0.025stake
 
-When everything runs successful, you will see the following output with a different account address:
+When everything runs successfully, you see the following output with a different account address:
```

Comment:
still building my blockchain and have to complete, but feel free to approve and merge to get better docs out there sooner :)

Reply:
Thanks, I will do this!